app/src/components/layout/pageTitle/PageTitle.tsx

```diff
@@ -0,0 +1,14 @@
+import { useEffect } from 'react';
+
+type Props = {
+    pageTitle: string;
+};
+
+function PageTitle({ pageTitle }: Props) {
+    const serviceName = 'Digital Lloyd George records';
+    useEffect(() => {
+        document.title = pageTitle + ' - ' + serviceName;
+    }, [pageTitle]);
+}
+
+export default PageTitle;
```

Comment:
Could we be clear about whether this function is intended as a hook function or a react component?
Usually, a component has Capital case naming, has a return value that render some html, and is used in the return part of other components
A hook function has the naming convention "useXxx", may not have a return value, and is used before the return part of other components
For now, the usage of this function looks like a hook, but it is put in the component folder, and there is a mix of `PageTitle` and `pageTitle` where it is referred.